jobs/src/main/scala/dcos/metronome/jobrun/JobRunService.scala

```diff
@@ -41,7 +41,7 @@ trait JobRunService {
     * @param jobSpec the specification to run.
     * @return the started job run
     */
-  def startJobRun(jobSpec: JobSpec, startingDeadline: Option[Duration] = None): Future[StartedJobRun]
+  def startJobRun(jobSpec: JobSpec, schedule: Option[ScheduleSpec] = None, startingDeadline: Option[Duration] = None): Future[StartedJobRun]
```

Comment:
if we go this way, we should get rid of startingDeadline as a separate field, because then it's also part of schedulespec